src/redux/actions/samples/loadSamples.js

```diff
@@ -1,22 +1,79 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
+import config from 'config';
+import { api } from 'utils/constants';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
+const toApiV2 = (samples) => {
+  const apiV2Samples = [{ samples: {} }];
+
+  const buildApiv1FilesObject = (fileObject) => {
+    const fileNames = [];
+    const apiV1FileObject = {};
+
+    Object.keys(fileObject).forEach((key) => {
+      const fileName = fileObject[key].s3Path.split('/').pop();
+      fileNames.push(fileName);
+
+      apiV1FileObject[fileName] = {
+        path: fileObject[key].s3Path,
```

Comment:
Just curious, what happens if we don't create the file with `path`? 

I think it isn't used anywhere anymore, so if we don't need it, it might be a good idea to assign `undefined` to it or something just to make it easy when we refactor the UI in the end.